static/jsonschema/kedro-catalog-0.18.json

```diff
@@ -829,9 +1086,61 @@
           }
         },
         {
-          "if": { "properties": { "type": { "const": "spark.SparkDataSet" } } },
+          "if": {
+            "properties": {
+              "type": {
+                "const": "redis.PickleDataSet"
+              }
+            }
+          },
           "then": {
-            "required": ["filepath"],
+            "required": [
+              "key"
+            ],
+            "properties": {
+              "key": {
+                "type": "string",
+                "description": "The key to use for saving/loading object to Redis."
+              },
+              "backend": {
+                "type": "string",
+                "description": "Backend to use, must be an import path to a module which satisfies the ``pickle`` interface.\nThat is, contains a `loads` and `dumps` function. Defaults to 'pickle'."
+              },
+              "load_args": {
+                "type": "object",
+                "description": "Pickle options for loading pickle files.\nHere you can find all available arguments:\nhttps://docs.python.org/3/library/pickle.html#pickle.loads\nAll defaults are preserved."
+              },
+              "save_args": {
+                "type": "object",
+                "description": "Pickle options for saving pickle files.\nHere you can find all available arguments:\nhttps://docs.python.org/3/library/pickle.html#pickle.dumps\nAll defaults are preserved."
+              },
+              "credentials": {
+                "type": "object",
+                "description": "Credentials required to get access to the redis server."
+              },
+              "redis_args": {
+                "type": "object",
+                "description": "Extra arguments to pass into the redis client constructor ``redis.StrictRedis.from_url``, as well as to pass to the ``redis.StrictRedis.set``"
+              },
+              "layer": {
+                "type": "string",
+                "description": "The data layer according to the data engineering convention:\nhttps://kedro.readthedocs.io/en/0.15.9/06_resources/01_faq.html#what-is-data-engineering-convention"
```

Comment:
This is a really old link, can we use a newer one? Preferable the one we'll have once 0.18.0 is out.